test/docker/docker-compose.yml

```diff
@@ -101,6 +100,7 @@ services:
     depends_on:
       - tigris_server
       - tigris_db2
+      - tigris_cache
```

Comment:
is this causing any issues for you? Shouldn't be needed as server2 is dependent on the server which waits for the cache.